libs/langchain-anthropic/package.json

```diff
@@ -35,7 +35,9 @@
   "author": "LangChain",
   "license": "MIT",
   "dependencies": {
+    "@anthropic-ai/bedrock-sdk": "^0.10.0",
     "@anthropic-ai/sdk": "^0.22.0",
+    "@anthropic-ai/vertex-sdk": "^0.4.0",
```

Comment:
Making this a required, rather than an optional, dependency adds a library for an edge case. (It also suggests that we can't include this in the google-vertex package, since we're not using this library there.)